apps/challenges/challenge_config_utils.py

```diff
@@ -506,6 +506,26 @@ def validate_challenge_config_util(
             )
             error_messages.append(message)
 
+        # Verify Tags are limited to 4
+        if "tags" in yaml_file_data:
+            tags_data = yaml_file_data["tags"]
+            if len(tags_data) > 4:
+                message = (
+                    "ERROR: Tags are limited to 4. Please remove extra tags"
+                    " then try again!"
+                )
+                error_messages.append(message)
+
+        # Verify Domain name is correct
+        if "domain" in yaml_file_data:
+            domain_value = yaml_file_data["domain"]
+            if domain_value not in ["CV", "NLP", "RL"]:
```

Comment:
I am not sure if this should be hardcoded.
Is there no way to take this list of domains from somewhere else?
Maybe import from models or somewhere?

Reply:
It is definitely possible,  I will change this part